lib/spack/spack/cmd/solve.py

```diff
@@ -70,7 +70,7 @@ def setup_parser(subparser):
         '--stats', action='store_true', default=False,
         help='print out statistics from clingo')
     subparser.add_argument(
-        '-T', '--together-where-possible', action='store_true', default=False,
+        '-T', '--unify-when-possible', action='store_true', default=False,
```

Comment:
This argument should be removed, we can just rely on the config option now (or make it a config-pushing option, like `--reuse`.

Reply:
Fixed in 08bc55e